README.md

```diff
@@ -35,7 +36,59 @@ TIZENRT_BASEDIR="$PWD"
 **Note**: To contribute in this community, you need to clone your forked private repository instead.  
           Github guides this by [working-with-forks](https://help.github.com/articles/working-with-forks).
 
-### 3. Configuration
+## How to build
```

Comment:
Numbering is required here.
For readability, naming convention is also required. For example, 
1. Installing Docker
2. Getting TizenRT source code
3. Building TizenRT
4. Programming